RecommendationEngine/Data/FacultyInterests.json

```diff
@@ -0,0 +1,2519 @@
+{
+    "Interests": [{
```

Comment:
```json
{
    "1": {
        "name": "Dr. Krishnashree Achuthan",
        "interest": ["Thin Films and Engineering of Materials for Electronic Applications"]
    },
    .....
}
```

Reply:
It's easier to iterate if the JSON is in that format.